spec/support/system_spec_helpers.rb

```diff
@@ -8,4 +8,8 @@ def row_with_content(content, element = page)
   def selected_govuk_tab
     find("div[class='govuk-tabs__panel']")
   end
+
+  def check_and_assess_list_item
```

Comment:
Is it worth making this dynamic / reusable ?

i.e. passing in the text as an arg so we can apply this to other list items

Reply:
```ruby
def list_item(text)
  find("li", text: text)
end
```

Reply:
Oh, excellent thought, yes! Especially because I've previously used the same thing elsewhere. DRY 🎉